frontend/apps/remark42/app/components/comment/comment-votes.tsx

```diff
@@ -38,8 +38,7 @@ export function CommentVotes({ id, votes, vote, disabled, controversy = 0 }: Pro
       setErrorMessage(undefined);
       setTimeout(() => setLoadingState(null), 200);
     } catch (err) {
-      // @ts-ignore
-      setErrorMessage(extractErrorMessageFromResponse(err, intl));
+      setErrorMessage(extractErrorMessageFromResponse(err as FetcherError, intl));
```

Comment:
Same here, we should check type of error before passing in into extract function like `err instanceof FectherError`